src/utils/config-helper.ts

```diff
@@ -51,6 +51,10 @@ export function fetchConfig (): Promise<any> {
       const keycloakConfigUrl = response.data['KEYCLOAK_CONFIG_URL']
       sessionStorage.setItem('KEYCLOAK_CONFIG_URL', keycloakConfigUrl)
       console.info('Set Keycloak config URL to: ' + keycloakConfigUrl)
+
+      const addressCompleteKey = response.data['ADDRESS_COMPLETE_KEY']
+      window['addressCompleteKey'] = addressCompleteKey
```

Comment:
most likely be a type error here. Maybe try `const someArbitraryWindowName = window as any` before assigning the completeKey value?

Reply:
Or you could ignore the lint error on the next line... something like:
`// eslint-disable-line no-console`
or
`// eslint-disable-next-line no-console`
(substitute the lint error accordingly)

Reply:
(<any>window).addressCompleteKey = addressCompleteKey seems to be working

Reply:
oh nice, i like that even more!